package.json

```diff
@@ -373,6 +373,11 @@
                                     "description": "Here you can give Calva some Clojure code to evaluate in the CLJ REPL, once it has been created.",
                                     "required": false
                                 },
+                                "projectRootDir": {
+                                    "type": "string",
+                                    "descripion": "The root directory to execute the Jack-in command from.  This will override the default behavior for determining the root directory relative to the current open file.  This is useful for working in monorepos.",
```

Comment:
FYI (and this should go to the dev docs): I've started to use `markdownDescription` to boost the schema help a bit.